pkg/datastore/documentloader/pdf.go

```diff
@@ -1,21 +1,21 @@
 package documentloader
 
 import (
+	"context"
+	"fmt"
+
 	"github.com/gptscript-ai/knowledge/pkg/datastore/types"
-)
 
-/*
- * Credits to https://github.com/hupe1980/golc/blob/main/documentloader/pdf.go
- */
+	/*
+	 * Credits to https://github.com/hupe1980/golc/blob/main/documentloader/pdf.go
+	 */
```

Comment:
nit: this comment should be moved to outside of the imports

Reply:
oh, thanks!